src/displayapp/DisplayApp.cpp

```diff
@@ -264,6 +275,10 @@ void DisplayApp::LoadApp(Apps app, DisplayApp::FullRefreshDirections direction)
         this, notificationManager, systemTask.nimble().alertService(), Screens::Notifications::Modes::Preview);
       ReturnApp(Apps::Clock, FullRefreshDirections::Up, TouchEvents::SwipeUp);
       break;
+    case Apps::Timer:
```

Comment:
Previous switch cases have braces around them, please add them here too for consistency.

Reply:
I don't see any of the surrounding cases having braces.

Reply:
`case Messages::TouchEvent: {`

Reply:
I see what you mean. GitHub collapses all the code between the changes. if you expand the path above this line, you see that in this particular section braces aren't used. Still kind of inconsistent. Probably a good idea to fix that in a dedicated PR.